packages/edgestitch/spec/dummy/config/initializers/application_controller_renderer.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+# Be sure to restart your server when you modify this file.
+
+# ActiveSupport::Reloader.to_prepare do
```

Comment:
Do we need these files?

Reply:
No… and I hate that rails generate them for us.